### ISSUE TITLE
Add support for "remote video" media in islandora_defaults.

### ISSUE THREAD

**Issue by jordandukart** [Islandora/documentation#1776]
As encountered by @bondjimbond in Slack there is currently no media type configuration set for remote video support within islandora_defaults. Given its inclusion [since 8.6](https://www.drupal.org/node/2966029) it would make sense to have this present on the sandbox and as a default option.

I am unsure of any sort of FCREPO ramifications with this but in terms of making it addable on [the route](https://github.com/Islandora/islandora/blob/8.x-1.x/src/Controller/ManageMediaController.php#L10-L34) it would be editing the media type to have a `field_media_of` field and exporting it to config.

**Comment by dannylamb** [comment]
We already do "external" datastreams with fcrepo for files that are stored on other filesystems, so I'm sure we can figure out how to make it work with fcrepo.  At least I wouldn't let it keep me up at night.  I'm sure there's a curveball in there somewhere, but it's manageable.

**Comment by kspurgin** [comment]
We have existing and potential clients who need/expect this. 

In all of these cases, there is descriptive metadata about the video content stored in Islandora. There is no expectation that the actual video file will be preserved in Fedora. 

With I7 clients we have supported this by using Basic image content model, having the Islandora-managed object file be some image usually a standard client logo, and having a link out to the external streaming video encoded in the MODS such that it displays prominently as a "View This Object" link. We do not have a external streaming video viewer going in I7

**Comment by dannylamb** [comment]
So I just took a youtube link and made a "Remote video" no problem.  It even snagged a thumbnail and set the title automagically.

![image](https://user-images.githubusercontent.com/20773151/111525072-efbea100-873b-11eb-864d-dca4d43f0e04.png)

I added the media of field and everything's fine, too.  This seems totally workable.

**Comment by jordandukart** [comment]
We're using this with a client already. Anything that is an oEmbed provider works and you can alter/add in providers that may not be in the standard listing as well.

**Comment by bondjimbond** [comment]
@kspurgin Just FYI for Islandora 7: https://github.com/bondjimbond/islandora_remote_media

**Comment by mjordan** [comment]
Should there be a "Media of" field on remote video? Here's what I'm seeing when I go to add one:

![image](https://user-images.githubusercontent.com/403918/113803864-418e9180-9712-11eb-9bab-e941ebf410c6.png)

**Comment by jordandukart** [comment]
Yep @mjordan that's part of what would need to be done to "support" this.

**Comment by mjordan** [comment]
2021-07-14 Tech Call: rough consensus was that we would treat embeded video (and eventually audio) like any other Islandora media (i.e. the media needs a `field_media_of` added to it). Exact process for getting this into the next release TBD.
